### PR TITLE
Add Code of conduct with slightly tweaked grammar

### PR DESCRIPTION
**Pull request by SolarDrew**

### CODE REVIEW

**Comment by Cadair** [comment]
I propose this becomes the canonical version of the CoC.

**Comment by nabobalis** [comment]
Yeah sure what evs

File: CODE_OF_CONDUCT.md [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+
+We recognise the work made by everyone and ensure the proper acknowledgement/citation of original authors at all times. As authors, we pledge to be explicit about how we want our own work to be cited or acknowledged.
+
+### Respectful
+
+Members of the community are respectful. We are respectful of others, their positions, their skills, their commitments, and their efforts. We are respectful of the volunteer and professional efforts within the community. We are respectful of the processes set forth in the community, and we work within them (paying particular attention to those new to the community).
+
+When we disagree, we are courteous in raising our issues. We provide a harassment- and bullying-free environment, regardless of sex, sexual orientation and/or gender identity, disability, physical appearance, body size, race, nationality, ethnicity, and religion. In particular, sexual language and imagery, sexist, racist, or otherwise exclusionary jokes are not appropriate. We will treat those outside our community with the same respect as people within our community.
```
Comment (nabobalis) [inline CODE_OF_CONDUCT.md]: Wouldn't both race and ethnicity be redundant?

Comment (Cadair) [inline CODE_OF_CONDUCT.md]: the dictionary seems to think they are subtly different.

Comment (SolarDrew) [inline CODE_OF_CONDUCT.md]: Yes, I would argue that they're different enough to merit individual mentions.

Comment (nabobalis) [inline CODE_OF_CONDUCT.md]: I concede. I have no issue with the PR.

File: CODE_OF_CONDUCT.md [hunk 2]
```diff
@@ -0,0 +1,31 @@
+The SunPy community is made up of members from around the globe with a diverse set of skills, personalities, and experiences. It is through these differences that our community experiences success and continued growth. We expect everyone in our community to follow these guidelines when interacting with others both inside and outside of our community. Our goal is to keep ours a positive, inclusive, successful, and growing community.
+
+A member of SunPy is:
+
+### Open
```
Comment (Cadair) [inline CODE_OF_CONDUCT.md]: line break after this title

**Pull request merged**